### PR TITLE
(#109) Separated search page

### PR DESCRIPTION
**Pull request by Patelzil**

### CODE REVIEW

File: frontend/src/components/searchBar.component.js [hunk 1]
```diff
@@ -95,24 +94,24 @@ class SearchBar extends React.Component {
...
-                            placeholder="500"
-                            value={this.state.maxPrice}
-                            onChange={this.handleMaxPrice}/>
-                    </FormGroup>{' '}
+                    {/*<FormGroup controlId="formInlineMin">*/}
+                    {/*    <FormLabel>Min Price</FormLabel>{' '}*/}
+                    {/*    <FormControl*/}
+                    {/*        type="number"*/}
+                    {/*        min = {0}*/}
+                    {/*        placeholder="0"*/}
```
Comment (patelp24-git) [inline frontend/src/components/searchBar.component.js:102]: It this thing needed or can be deleted?

File: frontend/src/components/searchBar.component.js [hunk 2]
```diff
@@ -66,17 +65,17 @@ class SearchBar extends React.Component {
...
-        this.setState({
-            maxPrice: event.target.value
-        })
-    }
+    // handleMinPrice = (event) => {
+    //     this.setState({
+    //         minPrice: event.target.value
+    //     })
+    // }
+    //
```
Comment (patelp24-git) [inline frontend/src/components/searchBar.component.js:73]: Same with this one?

**Review by patelp24-git — APPROVED** [review]

File: frontend/src/components/searchBar.component.js (hunk 1 above)
Comment (Patelzil) [reply frontend/src/components/searchBar.component.js:102]: can be uncommented later on

File: frontend/src/components/searchBar.component.js (hunk 2 above)
Comment (Patelzil) [reply frontend/src/components/searchBar.component.js:73]: same with this, can be uncommented later on

Comment (Rutu136) [reply frontend/src/components/searchBar.component.js:73]: sounds good! I will approve it!

**Review by Rutu136 — APPROVED** [review]

**Pull request merged**